### ISSUE TITLE
Strange behavior of DockWidget::show: growing in size 

### ISSUE THREAD

**Issue by amuralev** [KDAB/KDDockWidgets#133]
To reproduce the behavior you need two monitors:
1. Run example "kddockwidgets_example" on PRIMARY SCREEN;
2. Undock widget "DockWidget #0" (or any other) to FLOATING state;
3. MOVE mainwindow of application to SECONDARY SCREEN;
4. TOOGLE show/close of "DockWidget #0" from menu (for example about 10 times).

The floating window "DockWidget #0" will **grow in size** after each "DockWidget::show()"-call.

**Comment by iamsergio** [comment]
Which platform and which KDDW version ?

**Comment by amuralev** [comment]
Environment:
1. Windows 10 (64bit);
2. Qt 5.15.1 MSVC 2017 64bit;
3. KDDockWidgets 1.2 (1.1, RELEASE).

**Comment by iamsergio** [comment]
any HDPI scalling applied on the monitors, or they're both 100% ?

**Comment by amuralev** [comment]
No, scaling was not applied to any of the used monitors (they're both 100%).

**Comment by iamsergio** [comment]
reproduced

**Comment by iamsergio** [comment]
Not very confident about a fix... Can't see why having a transient parent on a different screen would influence the geometry.
Starts to sound either a Qt or a win32 bug

**Comment by iamsergio** [comment]
I've debugged this until reaching Qt internals, KDDW is setting the correct sizes, but there's a bug either in Qt Win32 QPA or Win32.
